.github/workflows/new-app-review.yml

```diff
@@ -15,8 +15,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-node@v4
         with:
+          # Fetch all history for all branches and tags
```

Comment:
This is a solution I found to the issue, but I can't really think of a way to test this out without having this workflows file merged in first. If I made this change on a forked repo branch I don't feel confident that it would work reliably.